skaha/src/main/java/org/opencadc/skaha/session/SessionAction.java

```diff
@@ -416,6 +421,8 @@ private List<Session> getSessions(String forUserID, String sessionID) throws Exc
                         } else {
                             instant = instant.plus(Integer.parseInt(expiryTimesStr), ChronoUnit.SECONDS);
                         }
+                        
+                        session.setExpiryTime(instant.toString());
```

Comment:
Looks like this will override the setting on <none> on line 420?

Reply:
Yes. When we do not find an expiry time for the uid, we set the expiry time to "none", otherwise we set the expiry time to the time that we calculated.

Reply:
But this line is called no matter what, so the "none" logic never sticks.

Reply:
Oh... right. Good eyes. I'll fix it. Thanks.